.Gruntfile.babel.js

```diff
@@ -256,7 +256,7 @@ module.exports = (grunt) => {
       output: {
         format: 'system',
         dir: distJS,
-        sourcemap: development,
+        sourcemap: development, // NOTE: change back to development before pushing PR
```

Comment:
This note is unclear

Reply:
Improved. Can you re-check?

Reply:
Sorry, I should have been clearer, what's unclear is the context. Just scanning the code doesn't explain why this comment is here. It's only clear to people who read the Troubleshooting section about sourcemap issues. So this needs to say something like "Change to `false` if experiencing sourcemap issues" with a link to the Troubleshooting document

Reply:
Recheck 2.0?

Reply:
No commit has been pushed?

Reply:
Yes it was, you need to check the new diff.